### DISCUSSION TITLE
dpgen run problem, how to using para_deg?

### DISCUSSION THREAD

**Discussion by llmxq** [deepmodeling/dpgen#1449] [Q&A]
i am now using bohrium platform. if i want to run two dp train jobs on one GPU, how do i set the parameters? i have tried to set para_deg = 2, but it didn't work. the following is the part of my machine.json file.
      "resources": {
        "number_node": 1,
        "cpu_per_node": 8,
        "gpu_per_node": 1,
        "para_deg": 2,
        "queue_name": "V100_8_32",
        "group_size": 1,
              "custom_flags": [],
              "strategy": {"if_cuda_multi_devices": true},
              "para_deg": 3,
              "source_list": []

**Comment by njzjz** [comment]
`group_size` should be no less than `para_deg`.
